### ISSUE TITLE
Add placeholder for %page% 

### ISSUE THREAD

**Issue by moonssons** [GC-spigot/battle-pass#493]
### Describe the feature

Please add a placeholder that shows what page a player is on in a menu. For example, in the quests GUI, I want to add an item inbetween the "Go Back" and "Next Page" buttons that shows which page the player is currently on

**Comment by vancorl** [comment]
Yes, I would really like to see a placeholder for the awards page.

**Comment by Intercrus** [comment]
good idea

**Comment by vancorl** [comment]
> Добавлен!

In the rewards menu, the placeholder always stays at value "1" !!
